WooCommerce/src/main/kotlin/com/woocommerce/android/ui/payments/changeduecalculator/ChangeDueCalculatorFragment.kt

```diff
@@ -10,18 +10,35 @@ import androidx.compose.ui.platform.ComposeView
 import androidx.fragment.app.viewModels
 import androidx.navigation.fragment.findNavController
 import com.woocommerce.android.ui.base.BaseFragment
+import com.woocommerce.android.ui.main.AppBarStatus
 import dagger.hilt.android.AndroidEntryPoint
 
 @AndroidEntryPoint
 class ChangeDueCalculatorFragment : BaseFragment() {
 
+    override val activityAppBarStatus: AppBarStatus
+        get() = AppBarStatus.Hidden
+
     private val viewModel: ChangeDueCalculatorViewModel by viewModels()
 
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        viewModel.navigationEvent.observe(
+            this
+        ) {
+            findNavController().navigateUp()
+        }
+    }
```

Comment:
A small suggestion regarding ViewModel Events: The `ScopedViewModel` type contains an event channel exposed through a LiveData instance alongside some canonical Event types, like the `Exit`. 

This allows you to do something like the following suggestion below without any new declaration or field:

```suggestion
        viewModel.event.observe(viewLifecycleOwner) { event ->
            when (event) {
                is Exit -> findNavController().navigateUp()
            }
        }
```

And then, the ViewModel function `onBackPressed` can just trigger the Event through the inherited Event channel:

```
fun onBackPressed() {
        triggerEvent(Exit)
    }
```

Reply:
I didn't know that. Thanks. I have changed that in this PR:
https://github.com/woocommerce/woocommerce-android/pull/11594